src/main/kotlin/com/cjbooms/fabrikt/generators/client/OpenFeignInterfaceGenerator.kt

```diff
@@ -53,6 +53,16 @@ class OpenFeignInterfaceGenerator(
 
             val clientType = TypeSpec.interfaceBuilder(simpleClientName(resourceName))
                 .addAnnotation(AnnotationSpec.builder(Suppress::class).addMember("%S", "unused").build())
+                .apply {
+                    if (options.contains(ClientCodeGenOptionType.SPRING_CLOUD_OPENFEIGN_STARTER_ANNOTATION)) {
+                        addAnnotation(
+                            OpenFeignAnnotations.feignClientBuilder()
+                                .addMember("configuration = %S", api.openApi3.info.getExtension("x-feign-client-name"))
```

Comment:
I've tried using vender options (x- prefixed), but any other suggestions?

Reply:
I think this approach is good. We also use it for Jackson annotations. But what happens is the options is turned on, but the extension is not present?

Reply:
This value should be the same for all Clients generated in a single fabrikt run, and the baseUrl, retryPolicy, etc. should be set based on this value.

It cannot contain whitespace, and lowerCamelCase is ideal.

Any suggestions for a good default value?

Reply:
If it's always the same, why not pass it in via a fabrikt config option, similar to:
` --http-model-suffix`

`x-` extensions make sense when the value needs to change in different parts of the spec, but it sounds like global external config is sufficient for you. 

I'm not sure about sensible defaults, I have never used this.

Reply:
Okay, it certainly sounds like a global external config would be better.

I'm on the road right now, so I'll make a commit to fix it when I get back.

Reply:
done.
It is now get as a cli option.